realm/realm-library/src/main/java/io/realm/internal/CheckedRow.java

```diff
@@ -126,7 +126,7 @@ public void setNull(long columnIndex) {
     protected native byte[] nativeGetByteArray(long nativePtr, long columnIndex);
     protected native int nativeGetMixedType(long nativePtr, long columnIndex);
     protected native Mixed nativeGetMixed(long nativeRowPtr, long columnIndex);
-    protected native long nativeGetLinkView(long nativePtr, long columnIndex);
+    public static native long nativeGetLinkView(long nativePtr, long columnIndex);
```

Comment:
When you modify `naive*` method as a `static`, don't forget to change the signature of corresponding JNI function arguments. The argument should be `(JNIEnv, jclass,...)` than `(JNIEnv, jobject,...)` could cause timeout on some devices. Look this guy's [comment](https://github.com/realm/realm-java/pull/2995#issuecomment-226070423)